android/sdl_android/src/main/java/com/smartdevicelink/managers/SdlManager.java

```diff
@@ -975,7 +975,7 @@ public SdlManager build() {
 			}
 
 			if (sdlManager.appId == null) {
-				throw new IllegalArgumentException("You must specify an app ID by calling setAppId");
+				throw new IllegalArgumentException("You must specify an app ID by calling setAppID");
```

Comment:
This is actually incorrect now; the method signature is setAppId() in the SdlManager